Modules/_statisticsmodule.c

```diff
@@ -0,0 +1,125 @@
+/* statistics accelerator C extensor: _statistics module. */
+
+#include "Python.h"
+#include "structmember.h"
+
+PyDoc_STRVAR(module_doc,
+"statisics speed up modules\n");
+
+
+static PyObject * normal_dist_inv_cdf(PyObject *self, PyObject *args);
+
+PyDoc_STRVAR(pydoc_normal_dist_inv_cdf,
+    "_normal_dist_inv_cdf(p, mu, sigma) |-> x\n"
+    "\n"
+    "Finds the value of the random variable such that the probability of\n"
+    "the variable being less than or equal to that value equals the given\n"
+    "probability.\n"
+    "\n"
+    "This function is also called the percent point function or quantile\n"
+    "function."
+);
+
+static PyMethodDef speedups_methods[] = {
+    {"_normal_dist_inv_cdf", normal_dist_inv_cdf, METH_VARARGS, pydoc_normal_dist_inv_cdf},
+    {NULL, NULL, 0, NULL}
+};
+
+
+static PyObject *
+normal_dist_inv_cdf(PyObject *self, PyObject *args)
+{
+    double p, mu, sigma, q;
+    double num, den, r, x;
+    if (!PyArg_ParseTuple(args, "d|d|d", &p, &mu, &sigma))
```

Comment:
Try using the ArgumentClinic on this.  It will likely make the function call even faster.

Here's my timings on the current version.

```
--baseline--
$ ./python.exe -m timeit -r11 -s 'from statistics import NormalDist as ND, quantiles as q' -s 'Z=ND()' 'q(Z, n=100)'
2000 loops, best of 11: 111 usec per loop

-- patched --
$ ./python.exe -m timeit -r11 -s 'from statistics import NormalDist as ND, quantiles as q' -s 'Z=ND()' 'q(Z, n=100)'
5000 loops, best of 11: 44 usec per loop
```

Reply:
As you suggested ArgumentClinic make better performance than I wrote.

Thanks!
````
5000 loops, best of 11: 40.6 usec per loop
````